core/state-machine/src/lib.rs

```diff
@@ -146,8 +146,14 @@ impl fmt::Display for ExecutionError {
 
 type CallResult<R, E> = Result<NativeOrEncoded<R>, E>;
 
-/// Externalities: pinned to specific active address.
-pub trait Externalities<H: Hasher> {
+/// Storage externalities API: pinned to specific active address.
```

Comment:
See the other comment below regarding the active address (unfortunately no link right now, since it is not possible to get a link while drafting a review)